packages/terra-button-group/tests/wdio/button-group-spec.js

```diff
@@ -120,6 +120,26 @@ Terra.describeViewports('Button Group', ['huge'], () => {
       browser.keys('ArrowLeft');
       Terra.validates.element('Button-1 focused');
     });
+
+    it('Navigates from first button to last button using left arrow', () => {
+      browser.keys('ArrowLeft');
+      Terra.validates.element('Last button focused');
```

Comment:
can we also programmatically check for focus? e.g. 
https://github.com/cerner/terra-core/blob/b635ece6c935b6004548684c03a881c9336c317a/packages/terra-search-field/tests/wdio/search-field-spec.js#L293

Reply:
Added programmatically check for focus